generators/entity-client/templates/angular/src/main/webapp/app/entities/update/entity-management-update.component.spec.ts.ejs

```diff
@@ -103,6 +103,22 @@ describe('Component Tests', () => {
                     expect(comp.isSaving).toEqual(false);
                 })
             );
+
+            it('Should cleanup on error',
+                fakeAsync(() => {
+                    // GIVEN
+                    const entity = new <%= entityAngularName %>(<%- tsKeyId %>);
+                    spyOn(service, 'update').and.returnValue(throwError('This is an error!'));
+                    comp.updateForm(entity);
```

Comment:
This should be `ngOnInit()` and data should come from `ActivatedRoute`.

Reply:
ActivatedRoute Mocking is not available on this branch.
We should switch to it on others tests too.